test/CalculateWinner.test.ts

```diff
@@ -0,0 +1,132 @@
+import { test } from "good-vibes";
+import { calculateWinnerV2 } from "../src/TiciTacaToeyGameEngine";
+
+test("Right Diagonal size 3 match", async (c) => {
+  c.log("Simple Check");
```

Comment:
lets remove this log statement as it was only for demonstration purposes

Reply:
sure,okay :)